README.md

```diff
@@ -1,9 +1,6 @@
 
 [![Code
-Climate](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/badges/2aa25d4736f09f40282e/gpa.svg)](https://codeclimate.com/repos/537da4a7e30ba062b101be9c/feed) [![Build
-Status](https://travis-ci.org/intercom/intercom-php.svg?branch=master)](https://travis-ci.org/intercom/intercom-php)
-
-## intercom-php
```

Comment:
Just double checking that this was removed on purpose. It was redundant so I'm happy either with or without it 😄